build.sh

```diff
@@ -134,6 +135,8 @@ function processCommandLineArgs {
         echo "  -n|--no-update    Don't update before building; just build."
         echo "  -l|--no-lint      Don't lint before building; just build."
         echo "  -h|--no-highlight Don't syntax-highlight the output."
+        echo "  -p|--single-page  Only build the single-page variant of the spec."
```

Comment:
I only did `--single-page` instead of `--single-page-only` because then our lines would get longer than 80 characters (after aligning all the others to the new width). Thoughts welcome.

Reply:
> I only did `--single-page` instead of `--single-page-only`

I think `--single-page` is better regardless